src/lib/layouts/workflow-header.svelte

```diff
@@ -1,4 +1,5 @@
 <script lang="ts">
+  import Badge from '$holocene/badge.svelte';
```

Comment:
Might have to change this import to `$lib/holocene/badge.svelte` according to @Alex-Tideman's PR: https://github.com/temporalio/ui/pull/1033